dasharo-compatibility/basic-platform-setup.robot

```diff
@@ -0,0 +1,67 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=30 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
+
+# TODO:
+# - document which setup/teardown keywords to use and what are they doing
+# - go threough them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keyword
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+BPS001.001 Power Control - Power On and Serial output
+    [Documentation]    Verifies if the DUT can be turned On and if the serial output can be read.
+    Power On
+    Sleep    60s
+    ${out}=    Read From Terminal
+    Should Not Be Empty    ${out}
+
+BPS001.002 Power Control - Power Off
+    [Documentation]    This test verifies if the DUT can be powered down.
+    Power On
+    Sleep    60s
+    ${out}=    Read From Terminal
+    Should Not Be Empty    ${out}
+    Power Cycle Off
```

Comment:
Does it include testing Power button, or only RTE relay / sonoff? We need to ensure both buttons and power supply operate correctly.